### ISSUE TITLE
Why are Region 23 values are always -nan in yolov3-tiny training log?

### ISSUE THREAD

**Issue by RanaOsamaAsif** [pjreddie/darknet#1612]
I'm facing an issue with training **yolov3-tiny**. I'm training the network to detect gender(2 custom classes) from face, the total size of my training dataset is 2000 face images(1000 for male & 1000 for female). Images are of mixed width and height most of them are small(192x256 or 128x192). I'm pretty sure that the images are all labeled correctly for ground truth BBs.

Ever since the training started values in the **Region 23** have been `-nan` except for `No Obj:` it always has a value. Everything else seems to work fine and average loss is steadily decreasing. 

Here's an example:
`Region 23 Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.00152, .5R: -nan, .75R: -nan, count: 0`

In the in the tiny cfg file I changed the number of `classes` to  2 and `filters` to 21 as per this formula filters=(classes + 5)*3. I also set the following `batch=24` and `subdivisions=8`, also changed the `width` and `height` to 256 from 416 since my images are small. The pretrained weights file I'm using is `yolov3-tiny.conv.15`. Didn't change anything else.

I'm training on CPU only the process is quite slow so far I have reached only 700 iterations. The avg. loss started from 625 now it's about 1.0672

I want to know are -nan values in Region 23 are normal or something is wrong?

**Comment by anilkasala** [comment]
Hi Rana, did you get any answer for this? Even I am getting the issue...

**Comment by aviksg** [comment]
Hi Rana, did you find any solution to this problem? Even I am facing the same issue

**Comment by raselldev** [comment]
> I'm facing an issue with training **yolov3-tiny**. I'm training the network to detect gender(2 custom classes) from face, the total size of my training dataset is 2000 face images(1000 for male & 1000 for female). Images are of mixed width and height most of them are small(192x256 or 128x192). I'm pretty sure that the images are all labeled correctly for ground truth BBs.
> 
> Ever since the training started values in the **Region 23** have been `-nan` except for `No Obj:` it always has a value. Everything else seems to work fine and average loss is steadily decreasing.
> 
> Here's an example:
> `Region 23 Avg IOU: -nan, Class: -nan, Obj: -nan, No Obj: 0.00152, .5R: -nan, .75R: -nan, count: 0`
> 
> In the in the tiny cfg file I changed the number of `classes` to 2 and `filters` to 21 as per this formula filters=(classes + 5)*3. I also set the following `batch=24` and `subdivisions=8`, also changed the `width` and `height` to 256 from 416 since my images are small. The pretrained weights file I'm using is `yolov3-tiny.conv.15`. Didn't change anything else.
> 
> I'm training on CPU only the process is quite slow so far I have reached only 700 iterations. The avg. loss started from 625 now it's about 1.0672
> 
> I want to know are -nan values in Region 23 are normal or something is wrong?

Me too, im create custom dataset for gender detection. Male and female. So far im ignore Region 23. And this my test dataset.
![image_output](https://user-images.githubusercontent.com/29136223/68542028-e9971280-03d9-11ea-970e-7f6e6427a0c1.jpg)
